app/src/main/java/dev/andrylat/app/App.java

```diff
@@ -69,18 +110,27 @@ public static void main(String[] args) {
             if (sum % 10 != 0) {
                 errors.append("-> Payment System can't be determined\n");
             }
+
+
+            
         }
+    }
 
+    public void print() {
         if (errors.length() > 0) {
             System.out.println("Card number is invalid.");
             System.out.println("Errors:");
             System.out.println(errors.toString());
         } else {
-            System.out.println("Card is valid.");
+            System.out.println(
+                new StringBuilder("Card is valid. Payment System is ")
+                    .append(brand)
+                    .toString()
+            );
         }
     }
 
-    private static boolean isDigit(String s) {
+    private boolean isDigit(String s) {
         if (s.equals(" ")) return false;
```

Comment:
Why PR doesn't contain whole code, only some changes? It is your first PR and it must contain whole code as new.

Reply:
Why you check only " "? Why not "  "? and "   " ?